src/fixit/config.py

```diff
@@ -156,26 +224,82 @@ def merge_configs(
 
     Assumes raw_configs are given in order from highest to lowest priority.
     """
-    kwargs: Dict[str, Any] = {
-        "root": None,
-    }
+
+    enable_rules: Set[str] = set()
+    disable_rules: Set[str] = set()
+    local_paths: List[Path] = []
+    rule_options: RuleConfigs = {}
+
+    def process_subpath(
+        subpath: Path,
+        *,
+        enable: Sequence[str] = (),
+        disable: Sequence[str] = (),
+        options: RuleConfigs = frozendict(),
+    ):
+        subpath = subpath.resolve()
+        if not path.is_relative_to(subpath):
+            print(f"ignoring {subpath}")
```

Comment:
this should either not be here, or converted into some kind of logging thing and suppressed by default